app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.raygun.raygun4androidsample.sample">
```

Comment:
We no longer have to define the package because of the namespace reference within `build.gradle` now?

Reply:
Yeah, it's essentially that. It's a change that came in with some version of Gradle 7.x:

https://developer.android.com/build/configure-app-module#set-application-id

Reply:
Awesome!